library/blivet.py

```diff
@@ -554,11 +557,27 @@ def _create(self):
                                                                                                                    parent.name,
                                                                                                                    parent.free_space))
 
+        create_vdo = self._blivet_pool._pool['vdo_deduplication'] or self._blivet_pool._pool['vdo_compression']
+        lv_size = size
+        if create_vdo:
+            try:
+                vdopool = self._blivet.new_lv(name='vdopool', vdo_pool=True,
+                                            parents=[parent], compression=self._blivet_pool._pool['vdo_compression'],
+                                            deduplication=self._blivet_pool._pool['vdo_deduplication'],
+                                            size=size)
+            except Exception as e:
+                raise BlivetAnsibleError("failed to set up volume '%s': %s" % (self._volume['name'], str(e)))
```

Comment:
Should we raise different Error log for the vdopool here and vdo lv below

Reply:
Fixed: Slightly modified message to improve future debugging.